GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -1071,7 +1141,9 @@ - (void)OAuthLoginWithOptions:(GIDSignInInternalOptions *)options
     [[[_authState expect] andReturn:tokenResponse] lastTokenResponse];
     [[[_authState expect] andReturn:tokenResponse] lastTokenResponse];
     if (oldAccessToken) {
+#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
```

Comment:
Why does this not happen on macOS?

Reply:
What was the resolution here?

Reply:
This is a step corresponding to EMM. Modified as iOS only.